source/read-write-concern/read-write-concern.rst

```diff
@@ -131,6 +136,14 @@ considered the server’s default ``ReadConcern``.
 default ``ReadConcern`` while the latter is the user explicitly specifying a
 ``ReadConcern`` with a ``level`` of “local”.
 
+Snapshot Read Concern
+---------------------
+
+When a ``ReadConcern`` ``level`` ``snapshot`` is used, ``atClusterTime`` MAY be specified to indicate
```

Comment:
MAY -> may

Capitalizing MAY here would indicate that a driver can decide whether or not to do it.  That's not what we're trying to say here.

Reply:
Done.